docs/YOSEMITE.md

```diff
@@ -36,6 +36,7 @@ That set of actions is declared in an implementation of the `Action` protocol. W
 
 At the moment, we provide the following subclasses of `Store`:
 * `AccountStore`. Registers `AccountAction` with the `Dispatcher`.  It implements the business logic necessary to manage an account (load an account, load a site, synchronise account information…)
+* `AppSettingsStore`. Registers and responds to actions declared in `AppSettingsAction`. It implements the logic to save and retrieve application settings.
```

Comment:
non-blocking & unrelated to your change: just noticed in line 46: `SettingAction` --> `ShipmentAction`